client/css/index.css

```diff
@@ -0,0 +1,26 @@
+a {
+  color: #099;
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+html {
+  height: 100%;
+  width: 100%;
+  font-family: 'Roboto', sans-serif;
+}
+
+body {
+  height: 100%;
+  width: 100%;
+  font-size: 13px;
+  line-height: 20px;
```

Comment:
Although in the [spec](http://www.google.com/design/spec/style/typography.html#typography-styles) desktop body text for CJK text (dense text) should be prefer 14px on desktop and 15px on mobile devices. I think font-size 14px or 15px is better here.

As for line height, the [spec](http://www.google.com/design/spec/style/typography.html#typography-line-height) suggests 23px, although I think 24px might be better because 24px aligns to a 4px grid.